pkg/bootstrap/config.go

```diff
@@ -274,6 +274,12 @@ func getStatsOptions(meta *model.BootstrapNodeMetadata) []option.Instance {
 		}
 	}
 
+	var compression string
+	// TODO: move annotation to api repo
+	if statsCompression, ok := meta.Annotations["sidecar.istio.io/statsCompression"]; ok {
```

Comment:
If we are adding a new annotation, there should be a release note to highlight it

Reply:
Sanity check, do we also need a proxy version check here?  Asking because I'm not sure if this is tied to any recent istio/proxy or recent commits in upstream envoyproxy repo.

Reply:
We should make it an environment variable. IMO this will not be a permanent API. We should:

1. Turn off by default with opt in
2. Turn on by default with opt-out
3. Remove opt-out, if no one needs it

If we are really confident I am ok with skipping (1).

Reply:
as @kyessenov suggest, we can exend to support zst or brotli, so maybe it can be a permanent API.